test/integration/singlecluster/scheduler/fairsharing/fair_sharing_test.go

```diff
@@ -66,15 +67,20 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		return cq
 	}
 
-	var createWorkload = func(queue string, cpuRequests string) *kueue.Workload {
+	var createWorkloadWithPriority = func(queue string, cpuRequests string, priority int32) *kueue.Workload {
 		wl := testing.MakeWorkloadWithGeneratedName("workload-", ns.Name).
+			Priority(priority).
 			Queue(queue).
 			Request(corev1.ResourceCPU, cpuRequests).Obj()
 		wls = append(wls, wl)
 		gomega.Expect(k8sClient.Create(ctx, wl)).To(gomega.Succeed())
 		return wl
 	}
 
+	var createWorkload = func(queue string, cpuRequests string) *kueue.Workload {
+		return createWorkloadWithPriority(queue, cpuRequests, 0)
```

Comment:
```suggestion
		return createWorkloadWithPriority(queue, cpuRequests, nil)
```

Acutally, the Workload priority field is a pointer. Should we distinguish 0 and null?

Reply:
I don't think it matters for this case. I only wanted to provide it so that there's the ability to specify different priorities.

Reply:
That makes sense. Indeed this `createWorkloadWithPriority` function is closed within fair_sharing testings file.